OtripleS.Portal.Web.Tests.Unit/Services/StudentViews/StudentViewServiceTests.Logic.RetrieveAll.cs

```diff
@@ -0,0 +1,83 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OtripleS.Portal.Web.Models.Students;
+using OtripleS.Portal.Web.Models.StudentViews;
+using Xunit;
+
+namespace OtripleS.Portal.Web.Tests.Unit.Services.StudentViews
+{
+    public partial class StudentViewServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveAllStudentsViewAsync()
+        {
+            // given
+            var randomUserId = Guid.NewGuid();
+            DateTimeOffset randomDateTime = GetRandomDate();
+
+            dynamic dynamicStudentProperties =
+                CreateRandomStudentViewProperties(
+                    auditDates: randomDateTime,
+                    auditIds: randomUserId);
+
+            var student = new Student
+            {
+                Id = dynamicStudentProperties.Id,
+                UserId = dynamicStudentProperties.UserId,
+                IdentityNumber = dynamicStudentProperties.IdentityNumber,
+                FirstName = dynamicStudentProperties.FirstName,
+                MiddleName = dynamicStudentProperties.MiddleName,
+                LastName = dynamicStudentProperties.LastName,
+                Gender = (StudentGender)dynamicStudentProperties.Gender,
+                BirthDate = dynamicStudentProperties.BirthDate,
+                CreatedDate = randomDateTime,
+                UpdatedDate = randomDateTime,
+                CreatedBy = randomUserId,
+                UpdatedBy = randomUserId
+            };
+
+            var studentView = new StudentView
+            {
+                IdentityNumber = dynamicStudentProperties.IdentityNumber,
+                FirstName = dynamicStudentProperties.FirstName,
+                MiddleName = dynamicStudentProperties.MiddleName,
+                LastName = dynamicStudentProperties.LastName,
+                Gender = (StudentViewGender)dynamicStudentProperties.Gender,
+                BirthDate = dynamicStudentProperties.BirthDate
+            };
+
+            var randomStudents = new List<Student> { student };
+            var actualStudentViews = randomStudents;
```

Comment:
As this entity is returned from the foundation, the name 'actual' does not suit well. How about 'returningStudents'?

Reply:
retrievedStudents?